rdr_service/model/participant_summary.py

```diff
@@ -56,6 +69,94 @@
 SUSPENDED_OR_DECEASED_PARTICIPANT_FIELDS = ["zipCode", "city", "streetAddress", "streetAddress2", "phoneNumber",
                                             "loginPhoneNumber", "email"]
 
+# SQL Conditional for participant's retention eligibility computed column (1 = NOT_ELIGIBLE, 2 = ELIGIBLE)
+# TO DO:  This should be updated to verify no deceased status when that is implemented (see: DA-1601)
+_COMPUTE_RETENTION_ELIGIBLE_SQL = """
+    CASE WHEN
+      consent_for_study_enrollment = 1
+      AND (consent_for_electronic_health_records = 1 OR consent_for_dv_electronic_health_records_sharing = 1)
+      AND questionnaire_on_the_basics = 1
+      AND questionnaire_on_overall_health = 1
+      AND questionnaire_on_lifestyle = 1
+      AND samples_to_isolate_dna = 1
```

Comment:
`samples_to_isolate_dna` could hold values of 10 or 11 to say that the samples have been DISPOSED or CONSUMED. Should a participant not be retention eligible if that's the case?

Reply:
Based on the requirement, only `samples_to_isolate_dna`=1 will be count as retention eligible.